### DISCUSSION TITLE
After BetterDisplay 3.1.2 I cannot change my display resolution to 720p

### DISCUSSION THREAD

**Discussion by mig8447** [waydabber/betterdisplaycli#9] [General]
Whenever I try to set it, nothing happens, it just exists with 0 and doesn't output "Failed."

**Comment by mig8447** [comment]
I tested by downgrading to 3.1.1 and it works ok

**Comment by waydabber** [comment]
What is the command?

**Comment by mig8447** [comment]
```sh
betterdisplaycli set -name='Home Center' -resolution=1280x720
```

**Comment by waydabber** [comment]
The command seems to be fine. What happens if you change the `set` to `get` - do you have your current resolution shown? If it is already 1280x720 with `set`, you should get a failed.

**Comment by mig8447** [comment]
Something interesting happened, I upgraded to 3.1.2 again (Downloading from releases instead of auto-update) and the command started working again, I'm not sure why.

**Comment by waydabber** [comment]
Hmm. Not sure why either. A quick simplification - you can use `-n=home` instead of `-name='Home Conter'` if you wish in commands like these to address the display (just reference some part of the name, case insensitive).
